### ISSUE TITLE
Change IRC nick and/or authenticate with Nickserv

### ISSUE THREAD

**Issue by Arthaey** [aeirola/slack-irc-client#6]
My Slack username matches my IRC nick, which I've registered with Nickserv. But when I start up slack-irc-client, I end up logged in to IRC as "arthaey1" instead of my normal "arthaey". How can I change my nick and/or authenticate with Nickserv?

**Comment by aeirola** [comment]
Does the console log anything regarding why the client was unable to use the specified nick? The slack-irc-client should try to connect to IRC with the Slack username as the nickname.

Haven't used nickserv myself, so can't say how it recognises users, and allows nick usage. If it uses the IRC username or real name you'll need to configure those in the server settings, which is configured as JSON in the "What I do field".

**Comment by Arthaey** [comment]
No, there's no messages from the bot in either the direct message window or in any of the channels. It just says that I'm connected as "arthaey1".

**Comment by aeirola** [comment]
Okay, then it sounds like the IRC server has just reserved the nick "arthaey" to some other client. This can probably be based on IP address, hostname, username, real name, or any other client identifier. It sounds more like an issue with the IRC server and/or Nickserv, unfortunately I can't help there.

**Comment by yokuyuki** [comment]
What I did to get around this was to open a private channel window with NickServ and sent `identify <pass>` to authenticate with NickServ on login.

Depending on what version of NickServ the IRC server is running, you can also identify and change your nick using `sidentify <nick> <pass>`.
